worker-pools.yml

```diff
@@ -1119,6 +1119,38 @@ pools:
                 - *scratch-disk
               machine_type: c2-standard-4
               capacityPerInstance: 1
+  - pool_id: '{pool-group}/images-gcp-aarch64'
```

Comment:
Should I add the `dind` stuff like in `images-gcp`? I didn't run into anything on try.

[This pull-request](https://github.com/taskcluster/docker-worker/pull/364) suggest using [these functions](https://github.com/taskcluster/taskcluster/blob/0edc545f46974be1b4ab11229bd61e3738e7d82f/workers/docker-worker/src/features/dind.js#L12) resolved the [issue](https://bugzilla.mozilla.org/show_bug.cgi?id=1427096).

Reply:
We shouldn't be using dind anywhere nowadays, so let's leave it out.